.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
 <!--
-Thank you for contributing to manim!
-
-Please ensure that your pull request works with the latest
-version of manim from this repository.
+Thanks for your contribution to ManimCommunity!
+
+Before filling in the details, ensure:
+- Your local changes are up-to-date with ManimCommunity/manim
+  
+- The title of your PR gives a descriptive summary to end-users. Some examples:
+  - `Fixed last animations not running to completion`)
+  - `Added gradient support and documentation for SVG files`)
+  Examples of what *NOT* to do:
+  - `fixed that styling issue` - not descriptive enough
+  - `fixed issue #XYZ` - end-user needs to do further research
```

Comment:
The content of this string is actually not markdown, no need for code literals there.

```suggestion
  - Fixed last animations not running to completion
  - Added gradient support and documentation for SVG files
  Examples of what *NOT* to do:
  - "fixed that styling issue" - not descriptive enough
  - "fixed issue #XYZ" - end-user needs to do further research
```